unst_msh_gen/README.md

```diff
@@ -45,5 +45,27 @@ NOTE: The output mesh will have -180:180 longitude, you can convert this by unis
 	output_file_path: shifted mesh in gmsh format with 0:360 long
 
 
+6- Using variable mode:
+	$python3 window_mask.py --windows '[{"min_lon": -98, "max_lon": -64, "min_lat": 24, "max_lat": 44.5, "hshr": 5}'
+
```

Comment:
Please add an example here or somewhere else in the code about how to define multiple windows.